### DISCUSSION TITLE
laravel-medialibrary 502 Bad Gateway

### DISCUSSION THREAD

**Discussion by yagogak** [filamentphp/filament#6054] [Help]
Hi,

I try to move from regular upload file ( for one of my resource ) to  SpatieMediaLibraryFileUpload.
The media is well copied in the disk, the record of the media is in the database also, everything seem to work , but when i'm redirected on the edit page i get a 502 Bad Gateway 
and the tail of nginx-error return
( *939 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1 )

**Comment by theavuth9** [comment]
Just me too  (:
